exporter/src/main/as/flump/export/texturepacker/SpaceSavingMultiPacker.as

```diff
@@ -0,0 +1,73 @@
+package flump.export.texturepacker {
```

Comment:
The space saving algorithm:  
1. Sort images by area and size. 
1. Find the biggest image, make an atlas that has the size of the next power of 2.
2. Try to pack every texture into it
3. if not all fit repeat from 1.